src/utils/yupSchema.ts

```diff
@@ -0,0 +1,26 @@
+import * as yup from 'yup';
+
+export const initialValuesFormik: FormValues = {
+  dishName: '',
+  preparationTime: '',
+  dishType: 'pizza',
+  noOfSlices: 1,
+  diameter: 20,
+  spicinessScale: 5,
+  slicesOfBread: 10,
+};
+
+export const yupSchemaCustomForm = yup.object({
```

Comment:
tu chyba to trzeba rozpisać, bo nie wszystkie pola są zawsze wymagane:
https://stackoverflow.com/questions/49394391/conditional-validation-in-yup